templates/workspace_services/ohdsi/README.md

```diff
@@ -0,0 +1,20 @@
+# OHDSI Workspace Service
+
+## IMPORTANT
+- This workspace service does not work "out of the box". It requires additional networking configuration to work properly. See the [Networking configuration](#networking-configuration) section for more details.
+- Currently the only data source supported by the workspace service is Azure Synapse.
+
+## Networking configuration
+Deploying the OHDSI workspace is not enough for it to function properly, in order for it to work properly, the following networking configuration should be in place:
+
+### 1. The resource processor should be able to access the data source
+Multiple OHDSI workspace services cannot share the same RESULTS and TEMP schemas because each OHDSI instance is changing the schemas, which could potentially cause conflicts.
+To avoid this, every workspace service must work on its own schemas. To do this, we use golden copying.
+This means that the "main" schemas remain untouched, and every workspace service has its own copy of the RESULTS and TEMP schemas, which it can modify.
+
+Since the resource processor is in charge of duplicating the schemas, the data source has to be accessible from the resource processor's VNet in order to be able to create them.
```

Comment:
So are we saying the resource processor needs to copy schema form the CDM data source to the in workspace DB?